docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Sphinx==1.3
-sphinxcontrib-napoleon
+Sphinx>=1.7,<2
+sphinx-rtd-theme
 releases
--e git://github.com/autoprotocol/autoprotocol-python.git@master#egg=autoprotocol
\ No newline at end of file
+-e git://github.com/autoprotocol/autoprotocol-python.git@master#egg=autoprotocol
```

Comment:
as far as I can tell these lines are the same. is this just git being weird?

Reply:
yea.. as far as I can tell, git is weird. I do not intend to change this line and manually examining the diff, it looks . the same